components/alert/AlertMap.vue

```diff
@@ -327,365 +402,446 @@ export default Vue.extend({
           // This does not immediately set the hex color, but instead waits for updateCutoffs to be called
-          region.faction = controlEvent.newFaction;
-          region.badge().update(this.map.getZoom());
-          this.updateLinks(controlEvent.facility);
+          region.faction = controlEvent.newFaction
+          region.badge().update(this.map.getZoom())
+          this.updateLinks(controlEvent.facility)
 
           // If we've reached the final region we're updating, apply the "captured" class to the hex to animate the color change
-          if(indexLimit && (index == this.captureIndices[indexLimit - 1]) || controlEvent.timestamp == lastCaptureEvent?.timestamp){
-            //var facility = (reverse && indexLimit) ? eventArray[this.captureIndices[indexLimit]].facility : controlEvent.facility;
-            var reverseFacility = (reverse && indexLimit) ? eventArray[this.captureIndices[indexLimit]].facility : undefined;
-            var facility = controlEvent.facility;
-            var region = this.mapRegions.get(facility)
-            if(!region){
-              console.error("Region not found for facility " + facility.toString());
-              return;
+          if (
+            (indexLimit && index === this.captureIndices[indexLimit - 1]) ||
+            controlEvent.timestamp === lastCaptureEvent?.timestamp
+          ) {
+            const reverseFacility =
+              reverse && indexLimit
+                ? eventArray[this.captureIndices[indexLimit]].facility
+                : undefined
+            const facility = controlEvent.facility
+            region = this.mapRegions.get(facility)
+            if (!region) {
+              console.error(
+                'Region not found for facility ' + facility.toString()
+              )
+              return
             }
-            var polygon = (<L.Polygon | undefined>this.polys.getLayer(region.outlineStamp));
-            polygon?.getElement()?.classList.add("captured");
-            polygon?.bringToFront();
+            const polygon = this.polys.getLayer(region.outlineStamp) as
+              | L.Polygon
+              | undefined
+            polygon?.getElement()?.classList.add('captured')
+            polygon?.bringToFront()
             setTimeout(() => {
-              polygon?.getElement()?.classList.remove("captured");
-            }, 1000);
-
-            if(reverseFacility){
-              region = this.mapRegions.get(reverseFacility);
-              if(!region){
-                console.error("Region not found for facility " + reverseFacility.toString());
-                return;
+              polygon?.getElement()?.classList.remove('captured')
+            }, 1000)
+
+            if (reverseFacility) {
+              region = this.mapRegions.get(reverseFacility)
+              if (!region) {
+                console.error(
+                  'Region not found for facility ' + reverseFacility.toString()
```

Comment:
Recommend the use of backticks for concatenation, imo it's cleaner:

```suggestion
                  `Region not found for facility ${reverseFacility.toString()}`
```

Reply:
Didn't even know that was a thing in JS, added it 👍

Reply:
Glad you discovered something 👍